frontend/src/features/properties/map/acquisition/AcquisitionContainer.tsx

```diff
@@ -78,12 +88,57 @@ export const AcquisitionContainer: React.FunctionComponent<IAcquisitionContainer
   const close = useCallback(() => onClose && onClose(), [onClose]);
 
   const onMenuChange = (selectedIndex: number) => {
-    setContainerState({ selectedMenuIndex: selectedIndex });
+    if (containerState.isEditing) {
+      if (formikRef?.current?.dirty) {
+        if (
+          window.confirm('You have made changes on this form. Do you wish to leave without saving?')
```

Comment:
This seems odd, why not use our styled modals here?

Reply:
was following the same logic/pattern that was set out in research file

Reply:
![image](https://user-images.githubusercontent.com/24322224/192915395-567d09b3-359c-4a0a-a770-b3e22bf3130e.png)
I figured it was there for a reason. Did not want to departure too much from pre-existing pattern/logic. But we can refactor both places next sprint if needed